libs/styles/styles.css

```diff
@@ -705,7 +705,6 @@ header:not(.global-navigation) ~ main {
   all of the ':not(.global-navigation)' rules can be entirely removed.
  */
 header.global-navigation {
-  height: var(--global-height-nav);
```

Comment:
I doubt this is what is desired, as removing this will increase CLS for all adobe.com pages. This change alone adds `0.03` CLS. Maybe that sounds negligible, but, per https://web.dev/articles/cls:
> To provide a good user experience, sites should strive to have a CLS score of 0.1 or less

So this change would essentially eat up 30% of the total CLS allowance on **all** our pages.

You likely don't want to tamper with this height. If the promo height should be different between viewports, you'd want to redefine `--global-height-navPromo` in https://github.com/adobecom/milo/blob/429001e956cdd9e4a36d4e8f6426c9e432ee5700/libs/styles/styles.css#L7 and adapt it based on media queries, to avoid CLS issues.

Reply:
@overmyheadandbody 
Added `global-navigation` height back to resolve that CLS issue.
Adjusted height `--global-height-navPromo` for feds promobar.